DigitalLearningSolutions.Data/DataServices/SelfAssessmentDataService/CandidateAssessmentsDataService.cs

```diff
@@ -29,7 +29,9 @@ public IEnumerable<CurrentSelfAssessment> GetSelfAssessmentsForCandidate(int del
                         CA.UnprocessedUpdates,
                         CA.LaunchCount,
                         1 AS IsSelfAssessment,
-                        CA.SubmittedDate
+                        CA.SubmittedDate,
+                        (SELECT CentreName FROM Centres
+                        WHERE CentreID=MAX(CA.CentreID)) AS CentreName

```

Comment:
So does this just return the most recent centre (name) only?

Reply:
Agree. I tend to think the query should join CandidateAssessments to Centres to retrieve the CentreName rather than using a subquery.

Reply:
The CentreName is now retrieved from Centres table